src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_prompt/index.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiText,
+  EuiHorizontalRule,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiPageContent,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { CreateButton } from '../create_button';
+
+export const CreateIndexPatternPrompt = ({
+  indexPatternCreationOptions
+}) => (
+  <EuiPageContent horizontalPosition="center">
+    <EuiEmptyPrompt
+      iconType="indexPatternApp"
+      title={
+        <EuiText grow={false}>
+          <h2>
+            <FormattedMessage id="kbn.management.indexPatternPrompt.title" defaultMessage="Create your first index pattern" />
+          </h2>
+        </EuiText>}
+      body={
+        <Fragment>
+          <p>
+            <FormattedMessage
+              id="kbn.management.indexPatternPrompt.subtitle"
+              defaultMessage="Index patterns allow you to bucket disparate data sources together so their shared fields may be queried in
+                Kibana."
+            />
+          </p>
+          <EuiHorizontalRule margin="l" />
+          <EuiText textAlign="left">
```

Comment:
I agree that this is helpful information and that it would be more effective in context of the index pattern creation flow. Placing it directly in the form would eat up quite a bit of real estate and interrupt more experienced users, so I'd recommend the flyout option.  

See the 'Small flyout' example in the EUI docs here: https://elastic.github.io/eui/#/layout/flyout

As for the trigger, an icon button (with the questionInCircle icon) in the top right of the 'Create index pattern' view would be consistent with the buttons on the index pattern detail view seen here: 

<img width="312" alt="screenshot 2019-02-11 15 28 14" src="https://user-images.githubusercontent.com/446285/52594739-d5ee5980-2e11-11e9-9635-72040ae1d4ed.png">

Reply:
@ryankeairns Do you think the content is small enough that a popover would be give enough room to show the info?

Reply:
It's fairly long and made the empty prompt even larger than I would have preferred. I think the skinny flyout feels like a good sized container for what is essentially 3 titles and paragraphs, in this case.